DIY-Peach!.md

```diff
@@ -0,0 +1,19 @@
+Guide for growing a Peach
+====
+
+As inspired by [Patterns for Decentralized Organizing](https://docs.wixstatic.com/ugd/8246de_248a47ddbd3c4d209667ff5508ee300b.pdf), [IDOARRT](https://toolbox.hyperisland.com/idoarrt-meeting-design), [9 Artifacts To Seed a Project Team](http://kfjournal.org/wp/index.php/2015/10/19/9-artifacts-to-seed-a-project-team/) and [Pods](https://handbook.enspiral.com/guides/pods.html) and [Stewardship](https://loomio.coop/stewarding.html) from Enspiral. 
```

Comment:
fyi i think this version of  Patterns for Decentralized Organizing is just a shorter pamphlet based on the longer book